core/includes/bootstrap.inc

```diff
@@ -1521,13 +1528,13 @@ function backdrop_load($type, $name) {
  * Note: When sending a Content-Type header, always include a 'charset' type,
  * too. This is necessary to avoid security bugs (e.g. UTF-7 XSS).
  *
- * @param $name
+ * @param string $name
  *   The HTTP header name, or the special 'Status' header name.
- * @param $value
+ * @param string|false $value
```

Comment:
I'm not sure about having `false` or `true` in `@param`. It might be allowed by our standards but I think I read somewhere how it can be set for the return value: `@return false` but not for `@param`. But now I can't find it or maybe it was just my impression. 

Maybe https://docs.phpdoc.org/3.0/guide/guides/types.html#supported-types?

Reply:
Maybe it's okay. I still haven't ready anything either way so I assume it might be ok.

https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#53-tags